libs/mrwebrtc-unityplugin/src/PlatformBase.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
This file and the ones in Unity are taken from https://github.com/Unity-Technologies/NativeRenderingPlugin AFAICS. By policy they either need to be in a separate directory (`external/NativeRenderingPlugin` or something like that) or they need to be annotated with the appropriate copyright notice.

Reply:
What copyright notice is needed? 

Would this be enough?
```
// Copyright (c) Unity Technologies.
// Licensed under the MIT License.
https://github.com/Unity-Technologies/NativeRenderingPlugin
```